packages/beacon-node/src/chain/blocks/index.ts

```diff
@@ -128,13 +128,15 @@ export async function processBlocks(
         const {signedBlock} = err;
         const blockSlot = signedBlock.message.slot;
         const {preState, postState} = err.type;
-        const forkTypes = this.config.getForkTypes(blockSlot);
-        const invalidRoot = toRootHex(postState.hashTreeRoot());
-
-        const suffix = `slot_${blockSlot}_invalid_state_root_${invalidRoot}`;
-        this.persistInvalidSszValue(forkTypes.SignedBeaconBlock, signedBlock, suffix);
-        this.persistInvalidSszView(preState, `${suffix}_preState`);
-        this.persistInvalidSszView(postState, `${suffix}_postState`);
+        const preRoot = preState.hashTreeRoot();
```

Comment:
~~`preRoot` and `postRoot` are only needed in the `catch` block. Should define them in there.~~

I guess at this point performance doesn't really matter